### ISSUE TITLE
rules_list: Group '5500' not found. Invalid 'if_group' with a fresh install 

### ISSUE THREAD

**Issue by wimstefan** [ossec/ossec-hids#1201]
Installing ossec from git on a Gentoo system gives me the following error when checking the config with "ossec-logtest"
`ossec-testrule: INFO: Reading local decoder file.
rules_list: Group '5500' not found. Invalid 'if_group'.`

Not sure why this happens on a clean install. Shouldn't it be possible to have a sane default rules configuration which enables one to fire up ossec immediately?

**Comment by ddpbsd** [comment]
It should be possible, just as it shpuld be possible to submit a polite and complete bug report.

**Comment by ddpbsd** [comment]
Fresh local installation using master from github:
```
 - The configuration can be viewed or modified at /var/ossec/etc/ossec.conf


    Thanks for using the OSSEC HIDS.
    If you have any question, suggestion or if you find any bug,
    contact us at contact@ossec.net or using our public maillist at
    ossec-list@ossec.net
    ( http://www.ossec.net/main/support/ ).

    More information can be found at http://www.ossec.net

    ---  Press ENTER to finish (maybe more information below). ---


ddp@ossec-local:~/ossec-hids$ /var/^C
ddp@ossec-local:~/ossec-hids$ sudo -s
sudo: unable to resolve host ossec-local
root@ossec-local:~/ossec-hids# /var/ossec/bin/ossec-logtest -t
2017/07/17 01:00:56 ossec-testrule: INFO: Reading local decoder file.
root@ossec-local:~/ossec-hids# echo $?
0
root@ossec-local:~/ossec-hids# /var/ossec/bin/ossec-control start
Starting OSSEC HIDS v2.9.0 (by Trend Micro Inc.)...
2017/07/17 01:01:05 ossec-maild: INFO: E-Mail notification disabled. Clean Exit.
Started ossec-maild...
Started ossec-execd...
Started ossec-analysisd...
Started ossec-logcollector...
Started ossec-syscheckd...
Started ossec-monitord...
Completed.
```

**Comment by wimstefan** [comment]
My apologies if I sounded rude. That was the last of my intentions! Would you please be so kind and point me to the documentation for a correct bug report? The only thing I found was this on https://ossec.github.io/about.html#support-options:

> OSSEC Github
> You can post issues and get caught up on OSSEC development at the OSSEC Github account.

That's what I tried to do but obviously failed ;-)

The steps I followed were:
1. `git clone  https://github.com/ossec/ossec-hids`
2. Installing with the provided install.sh script using the default configuration offered during the installation process
3. Starting the app with `/var/ossec/bin/ossec-control start`
This gives the following error:
>Starting OSSEC HIDS v2.9.0 (by Trend Micro Inc.)...
>ossec-analysisd: Configuration error. Exiting.
4. `/var/ossec/bin/ossec-logtest -t` has this output:
>ossec-testrule: INFO: Reading local decoder file.
>rules_list: Group '5500' not found. Invalid 'if_group'.

I hope this is already a little bit better. Do you want me to paste the content of /var/ossec/etc/ossec.conf and /var/ossec/rules/pam_rules.xml as well?

**Comment by ddpbsd** [comment]
Specifying the version of OSSEC would have been a huge benefit. I could assume you used MASTER, but that's not the best option. Also which OS or distribution you're using. We would need to figure out what your environment is, because this isn't the behavior we're seeing in our environments.

Maybe the output of `grep 'if_group>5500' *` would be helpful.

**Comment by wimstefan** [comment]
I see. I presumed "Installing ossec from git on a Gentoo system" would be enough information for a start. My apologies again. 

But I guess I resolved the problem anyway . I wanted to reproduce the whole installation process in a script session and therefore deleted not only the `/var/ossec/` folder but also the local git repository that I cloned back in 2015 and updated regularly since then. After cloning the git tree into a fresh local repository I could not only finish the installation but also start ossec without errors.

Thank you for your patience :)
